cli/Valet/Site.php

```diff
@@ -190,6 +190,29 @@ public function proxies()
         return $proxies;
     }
 
+    /**
+     * Determine if the provided site is a valid site, whether parked or linked and get the site url.
+     *
+     * @param  string  $directory
+     * @return string|false
+     */
+    public function getSiteUrl($directory)
+    {
+        $tld = $this->config->read()['tld'];
+
+        if ($directory == '.') { // Allow user to use dot as current dir's site `--site=.`
+            $directory = $this->host(getcwd());
+        }
+
+        $directory = str_replace('.'.$tld, '', $directory); // Remove .tld from sitename if it was provided
+
+        if ($this->parked()->merge($this->links())->where('site', $directory)->count() > 0) {
+            return $directory.'.'.$tld;
+        }
+
+        return false; // Invalid directory provided
```

Comment:
Let's flip these so we end on the happy path:

```suggestion
        if (! $this->parked()->merge($this->links())->where('site', $directory)->count() > 0) {
            return false; // Invalid directory provided
        }

        return $directory.'.'.$tld;
```